app/services/oa_workflow_service.rb

```diff
@@ -33,5 +33,7 @@ def workflow
     rescue StandardError
       file.update_column(:downloaded, false)
     end
+
+    AiOAStatusExportJob.perform_later
```

Comment:
Sorry, I should have caught this before.  I think it would be best to iterate through all the files that need their status sent to Activity Insight here, and then kick off the jobs one-by-one.  So the `ActivityInsightOAStatusExporter` would initialize with a single file rather than all the files.  Then, I think we'd want delayed_job to just handle the errors for us.  I'll describe that below.  Overall, I think doing things one-by-one will make troubleshooting and tracking jobs easier and more organized.